<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDateEdit" name="d_from">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="d_to">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>с</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_contract">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Договор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Выдать пропуск</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="d_note">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Служебка от:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Исх.:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_open">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
